packages/headless/src/api/analytics/search-analytics.test.ts

```diff
@@ -1,10 +1,10 @@
 import {CoveoAnalyticsClient} from 'coveo.analytics';
 import pino from 'pino';
-import {DateFacetValue} from '../../controllers/product-listing/range-facet/date-facet/headless-product-listing-date-facet';
 import {getConfigurationInitialState} from '../../features/configuration/configuration-state';
 import {getCategoryFacetSetInitialState} from '../../features/facets/category-facet-set/category-facet-set-state';
 import {getFacetSetInitialState} from '../../features/facets/facet-set/facet-set-state';
 import {FacetSortCriterion} from '../../features/facets/facet-set/interfaces/request';
+import {DateFacetValue} from '../../features/facets/range-facets/date-facet-set/interfaces/response';
```

Comment:
What's the link of this with the product listing?

Reply:
we use to import the interface DateFacetValue directly from a deleted headless product listing date facet. This was just wrong and probably a vs code autocomplete